examples/nextjs/src/createEmotionCache.js

```diff
@@ -0,0 +1,9 @@
+import createCache from '@emotion/cache';
+
+export default function createEmotionCache() {
+  const cache = createCache({ key: 'css' });
+  // compat is important. It prevents emotion to render invalid HTML.
+  // See https://github.com/mui-org/material-ui/issues/26561
+  cache.compat = true;
+  return cache;
```

Comment:
```suggestion
  return createCache({ key: 'css' });
```

Reply:
Let's move the comment here:

```diff --git a/examples/nextjs/pages/_document.js b/examples/nextjs/pages/_document.js
index e562aeec4a..e712dc012e 100644
--- a/examples/nextjs/pages/_document.js
+++ b/examples/nextjs/pages/_document.js
@@ -72,6 +72,8 @@ MyDocument.getInitialProps = async (ctx) => {
     });

   const initialProps = await Document.getInitialProps(ctx);
+  // This is important. It prevents emotion to render invalid HTML.
+  // See https://github.com/mui-org/material-ui/issues/26561
   const emotionStyles = extractCriticalToChunks(initialProps.html);
   const emotionStyleTags = emotionStyles.styles.map((style) => (
     <style
```

as per https://github.com/mui-org/material-ui/issues/26957#issuecomment-880191571.